docs/source/metadata.rst

```diff
@@ -8,29 +12,67 @@ or an e-mail with a given subject line.
+|                             | for max_characters in add_chunking_strategy              |                                                                                                                                                                                                                                                                                             |
++-----------------------------+----------------------------------------------------------+---------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------+
+| ``is_continuation``         | True if element is a continuation of table being chunked | Used for chunking                                                                                                                                                                                                                                                                           |
++-----------------------------+----------------------------------------------------------+---------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------+
+| ``detection_class_prob``    | Detection Model Class Probabilities                      | From unstructured-inference, hi-res strategy                                                                                                                                                                                                                                                |
++-----------------------------+----------------------------------------------------------+---------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------+
+
+:raw-html:`<br />`
+Notes on common metadata fields:
+
+Coordinates
+-----------
 
 Some document types support location data for the elements, usually in the form of bounding boxes.
 If it exists, an element's location data is available with ``element.metadata.coordinates``.
 
 The ``coordinates`` property of an ``ElementMetadata`` stores:
 
-* points: These specify the corners of the bounding box starting from the top left corner and
+* **points**: These specify the corners of the bounding box starting from the top left corner and
```

Comment:
Points should probably be monospace font, since it refers to a keyword in the code:
```suggestion
* ``points`` : These specify the corners of the bounding box starting from the top left corner and
```